managing_cluster_configuration/managing-openshift-cluster-configuration.adoc

```diff
@@ -0,0 +1,46 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="managing-openshift-cluster-configuration"]
+include::_attributes/common-attributes.adoc[]
+= Managing {OCP} cluster configuration
+
+:context: managing-openshift-cluster-configuration
+
+toc::[]
+
+With {gitops-title}, you can manage your {OCP} cluster configuration to have the following benefits:
+
+* Version control and auditability: Configuration changes committed to Git provides a complete history of modifications. This facilitates auditing, compliance, and accountability.
+* Single source of truth: Git serves as the definitive source for the desired state of your {OCP} cluster.
+* Optimized performance and disaster recovery: {gitops-shortname} points the Argo CD application to the previous commit or tag with a known-good state in Git, which in turn reduces downtime and helps with disaster recovery.
+* Collaboration and Review: Git's collaborative features enable team members to review and approve infrastructure and application configuration changes before they are applied to your {OCP} cluster.
+* Efficiency and Scalability: {gitops-shortname} streamlines the deployment and operations workflows, enabling efficient management of complex and multi-cluster environments with reduced manual intervention and human error.
+
+Perform the following tasks to manage your {OCP} cluster configuration:
+
+. Installing the {gitops-title} Operator using CLI
+. Analyzing the default Argo CD instance
+. Accessing the default Argo CD instance
+. Configuring the default Argo CD instance
+
+// Installing Red Hat OpenShift GitOps Operator using CLI
+include::modules/installing-gitops-operator-using-cli.adoc[leveloffset=+1]
```

Comment:
Seems like we missed adding gitops-plugin pod there

Reply:
Updated the example output